thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/alert/EmailHelper.java

```diff
@@ -121,4 +127,39 @@ public static DateTime calculateGraphDataStart(DateTime start, DateTime end,
     }
     return start;
   }
+
+  public static void sendEmailWithTextBody(HtmlEmail email, SmtpConfiguration smtpConfigutation, String subject,
```

Comment:
This function looks the same as the sentEmailWithHtml defined in line 137. Can you consolidate these two?